dotCMS/src/integration-test/java/com/dotmarketing/webdav/DotWebdavHelperTest.java

```diff
@@ -1,5 +1,7 @@
 package com.dotmarketing.webdav;
```

Comment:
Again, should this be part of the PR?

Reply:
I recently introduced this method `upperCaseRandom` when I worked on webdav for this new work I needed to test casing so I relocated the method so it could be reused everywhere